comit/src/expiries/config.rs

```diff
@@ -0,0 +1,289 @@
+            ActorType::Human => 60.minutes(),
+        }
+    }
+}
+
+impl Config {
+    // Optimisation: finality_alpha/beta never change, we should cache the value so
+    // we only do the calculation once.
+
+    // Open Question: Is time_to_mine_n_blocks() _really_ better than just doing:
+    //
+    //   `AVERAGE_BLOCK_TIME * NUMBER_OF_CONFIRMATIONS`
+
+    /// The duration of time it takes for Alice to act.
+    pub fn alice_to_act(&self) -> Duration {
+        self.alice.period_to_act()
+    }
+
+    /// The duration of time it takes for Bob to act.
+    pub fn bob_to_act(&self) -> Duration {
+        self.bob.period_to_act()
+    }
+
+    /// The duration of time it takes for the alpha fund transaction to be
+    /// mined into the blockchain.
+    pub fn mine_alpha_fund_transaction(&self) -> Duration {
+        let n = self.alpha_mine_fund_within_n_blocks;
+        let block_time = self.alpha_block_time;
+
+        time_to_mine_n_blocks(n, block_time)
+    }
+
+    /// The duration of time it takes for the beta fund transaction to be
+    /// mined into the blockchain.
+    pub fn mine_beta_fund_transaction(&self) -> Duration {
+        let n = self.beta_mine_fund_within_n_blocks;
+        let block_time = self.beta_block_time;
+
+        time_to_mine_n_blocks(n, block_time)
+    }
+
+    /// The duration of time it takes for the alpha redeem transaction to be
+    /// mined into the blockchain.
+    pub fn mine_alpha_redeem_transaction(&self) -> Duration {
+        let n = self.alpha_mine_redeem_within_n_blocks;
+        let block_time = self.alpha_block_time;
+
+        time_to_mine_n_blocks(n, block_time)
+    }
+
+    /// The duration of time it takes for the beta redeem transaction to be
+    /// mined into the blockchain.
+    pub fn mine_beta_redeem_transaction(&self) -> Duration {
+        let n = self.beta_mine_redeem_within_n_blocks;
+        let block_time = self.beta_block_time;
+
+        time_to_mine_n_blocks(n, block_time)
+    }
+
+    /// The duration of time it takes for a transaction to reach finality on the
+    /// alpha ledger.
+    #[allow(clippy::cast_possible_truncation, clippy::cast_possible_wrap)] // Block times should not overflow an i32.
+    pub fn finality_alpha(&self) -> Duration {
+        let n = self.alpha_required_confirmations;
+        let block_time = self.alpha_block_time;
+
+        time_to_mine_n_blocks(n, block_time)
+    }
+
+    /// The duration of time it takes for a transaction to reach finality on the
+    /// beta ledger.
+    #[allow(clippy::cast_possible_truncation, clippy::cast_possible_wrap)] // Block times should not overflow an i32.
+    pub fn finality_beta(&self) -> Duration {
+        let n = self.beta_required_confirmations;
+        let block_time = self.beta_block_time;
+
+        time_to_mine_n_blocks(n, block_time)
+    }
+}
+
+fn bitcoin_required_confirmations() -> usize {
+    6 // This is the accepted value within the Bitcoin ecosystem.
```

Comment:
Maybe worth adding those as constants at the start of the file.

Reply:
Will do, thanks for prodding me.